modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedMessageStoreImpl.java

```diff
@@ -549,11 +507,11 @@ public void moveMetaDataToQueue(long messageId,
             throw new AndesException("Message MetaData not found to move the message to Dead Letter Channel");
         }
 
-        List<AndesRemovableMetadata> removableMetadataList = new ArrayList<AndesRemovableMetadata>(1);
-        removableMetadataList.add(new AndesRemovableMetadata(messageId, currentQueueName, currentQueueName));
+        List<Long> removableMetadataList = new ArrayList<Long>(1);
```

Comment:
The method 'deleteMessageMetaDataFromQueue(String storageQueueName, List<Long> messagesToRemove)' accepts an arrayList of messageIDs. But we only make the call for one messageID. Therefore, an arrayList of capacity 1 is created and the message ID is added. This is how it was done from earlier. Could add a separate method overloading the original to delete a single message. But looking at the usages, it will only be used in Cassandra. Should I add it?